agent/php_user_instrument.h

```diff
@@ -177,7 +189,8 @@ extern nruserfn_t* nr_php_add_custom_tracer_callable(
     zend_function* func TSRMLS_DC);
 extern nruserfn_t* nr_php_add_custom_tracer_named(const char* namestr,
                                                   size_t namestrlen,
-                                                  nr_transience_t transience TSRMLS_DC);
+                                                  nr_wrap_user_function_options_t options
```

Comment:
Should it be passed by value or by reference? Also nr_php_add_custom_tracer_named does not modify options. If that's the case shouldn't this be indicated with a `const`?

Reply:
fixed in [449afaa](https://github.com/newrelic/newrelic-php-agent/pull/730/commits/449afaacda9a9d734919944b3742d90d348eb3d9)